src/quo/components/bottom_sheet/style.cljs

```diff
@@ -24,8 +25,12 @@
    :bottom           0})
 
 (defn content-container
-  [window-height]
-  {:background-color        (:ui-background @colors/theme)
+  [window-height new-ui?]
+  {:background-color       (if new-ui?
+                             (quo2.colors/theme-colors
+                              quo2.colors/neutral-10
+                              quo2.colors/neutral-80)
+                             (:backdrop @colors/theme))
```

Comment:
hi @jo-mut, when `new-ui?` will be false it will use `(:backdrop @colors/theme)` as `:background-color` instead of `(:ui-background @colors/theme)`. Wouldn't that will affect old ui?